### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by IzzySoft** [ryuunoakaihitomi/ForceCloseLogcat#5]
Could you please state what license you are using for this project?

**Comment by ryuunoakaihitomi** [comment]
@IzzySoft What are you going to do with this project?

I will choose an appropriate license for this in the future. But now ["No License"](https://choosealicense.com/no-permission/) can temporarily be used as a license for the project.

**Comment by IzzySoft** [comment]
Currently, I'm providing the app [in my repository](https://apt.izzysoft.de/fdroid/index/apk/com.ryuunoakaihitomi.ForceCloseLogcat). It seems your app would be eligible for the "official [F-Droid](https://f-droid.org/) repo – so if you wish, I could help you bringing it in. But the latter needs an OSI approved license (in my repo it's nice-to-have, but in the official its a must).

Following up your link: yes, that's what I'm currently doing: "Ask the maintainers nicely to add a license" :smile:

**Comment by ryuunoakaihitomi** [comment]
@IzzySoft Is that OK?I am not sure.

However, I don't recommend that you release it in the F-Droid repo. The code quality of this app is not high and the stability is poor (because of logcat, I may not be able to improve this point).

But if you like,you can do all the things you want to do.

**Comment by IzzySoft** [comment]
Yes, that's excellent! I've updated it on my end, so it should show up after the next sync in a few hours. Thanks!

And if you don't want your app in the official repo, we of course respect that (I've made a note in case someone files a request; disclosure: I'm one of the F-Droid maintainers). Just let me know (e.g. by updating this issue here which I'm following) if you changed your mind and want it included in the official repo, I'd then gladly help you achieving that goal.

Thanks again!

**Comment by IzzySoft** [comment]
One more question: will the be a version with English locale?

**Comment by ryuunoakaihitomi** [comment]
@IzzySoft 

I **will** translate its text resources into English,but not now.

I have been very busy for the past few months,and there will be a lot of trifling things waiting for me in the future.As a non-native English speaker,it's not very easy to translate the help text into concise English.(Unfortunately, even for my native language,I am not very good at expressing)

**Comment by IzzySoft** [comment]
Thanks, and understood! As long as it is "on the horizon", it gives hope; just a "no" or "unlikely" would be disappointing :smile: So whenever you're ready it will be there – which is cool!
